Sources/PubNub/Networking/Routers/ObjectsChannelRouter.swift

```diff
@@ -10,13 +10,17 @@
 
 import Foundation
 
+extension Array where Element == ObjectsChannelRouter.Include {
+  func values() -> String {
+    return map { $0.rawValue }.csvString
+  }
+}
+
 public struct ObjectsChannelRouter: HTTPRouter {
   public enum Endpoint: CustomStringConvertible {
-    case all(
-      customFields: Bool, totalCount: Bool, filter: String?, sort: [String], limit: Int?, start: String?, end: String?
-    )
-    case fetch(metadataId: String, customFields: Bool)
-    case set(metadata: PubNubChannelMetadata, customFields: Bool)
+    case all(include: [Include]?, totalCount: Bool, filter: String?, sort: [String], limit: Int?, start: String?, end: String?)
```

Comment:
Isn't `totalCount` part of inclusion flags? Maybe it will make sense to merge into `Include`?

Reply:
Ok, let's keep it as is for now.